### PR TITLE
Add tarot boosted apr

### PR DESCRIPTION
**Pull request by groninge01**
image

I added bignumber.js to make the conversion of the apr less funky.

### CODE REVIEW

**Review by danielmkm — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
         "apollo-server-core": "^3.5.0",
         "apollo-server-express": "^3.5.0",
         "axios": "^0.24.0",
+        "bignumber.js": "^9.0.2",
```
[the remark this answers is not in the document]
Comment (groninge01) [reply package.json]: @danielmkm I have removed bignumber.js and have used the ethers version. I also added xtinspirit & xtarot.

File: modules/boosted/tarot.service.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+}
+
+const TAROT_APR_CONTRACT_ADDRESS = '0xc32B9a98F2419544070E9580166C5A18Af8ADf9f';
+const tarotAPRContract = getContractAt(TAROT_APR_CONTRACT_ADDRESS, TarotBoostedAbi);
+
+export class TarotService {
+    private cache: CacheClass<string, TarotData[]>;
+    private boostedPools = [
+        {
+            address: '0xfB98B335551a418cD0737375a2ea0ded62Ea213b', // MAI
```
Comment (danielmkm) [inline modules/boosted/tarot.service.ts:24]: don't hardcode these addresses like this.
- Use the factory address on the pool to determine its a tarot pool
- Use the `mainIndex` and `wrappedIndex` on the pool to determine main/wrapped tokens.

**Review by danielmkm — CHANGES_REQUESTED** [review]

File: modules/boosted/tarot.service.ts (hunk 2 above)
Comment (groninge01) [reply modules/boosted/tarot.service.ts]: I have hardcoded the factory address (for now). Is there a way to get it from the balancer sdk config? Do I need to resolve the 'network' key somehow first to get to the 'linearPools' key?

Comment (groninge01) [reply modules/boosted/tarot.service.ts]: @danielmkm I figured out how to grab the factory address from the balancer sdk so it is no longer hardcoded anymore.

**Pull request merged**